### ISSUE TITLE
The limits of our Savage-Dickey BF estimate

### ISSUE THREAD

**Issue by mattansb** [easystats/bayestestR#152]
Reading into the theory and practicality of the sdBF I've found some points regarding the limits of what we offer with out implementation (I'm writing this here to organize my thoughts - we can see if or how these points should be addressed in and of the docs):

**Point 1 - Theory**: As [Heck (2018)](https://doi.org/10.1111/bmsp.12150) points out, equating the Savage-Dickey density ratio (SDR) to a BF (model vs null model) is only *true* under some conditions regarding the nusiance parameters which are hardly ever met.
**Counterpoint 1**:
1. The SDR is still a good approximation of the true BF (using the examples in the paper, r > 0.8, and most importantly in the same direction).
2. In tests I've done, the largest deviation from other BF methods (integration as in `BayesFactor` or bridge sampling with `rstanarm`/`brms`+`bridgesampling`) was never larger than a factor of 2, and was usually nearly identical.
3. The suggested correction (generalized sdBF) is computationally expensive - might as well compute a "real" (marginal likelihood) BF.
4. The SDR has *theoretical meaning by itself* - it quantifies how much more / less probable the null is in light of the observed data.

**Point 2 - practical / computation**: [Morey et al. (2011)](https://doi.org/10.1016/j.jmp.2011.06.004) point out the sdBF estimates that are based on splines of kernels (what we offer) are prone to error.
**Counterpoint 2**:
1. See point 2 above.
2. Other methods are again more computationally expensive.

In conclusion, the spline/kernel based SDR should be used as it is simple to compute, and conveys inherit statistical meaning even when is isn't exactly representing a true BF.

FIN.

**Comment by DominiqueMakowski** [comment]
Interesting, thanks for sharing this and for the summary :)

**Comment by mattansb** [comment]
Closing this - it can still be searched, but I think the vignette covers fully what the Savage-Dickey BF is and is not.
